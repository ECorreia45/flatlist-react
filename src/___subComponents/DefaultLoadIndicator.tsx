import React from 'react';

const DefaultLoadingIndicator = (): JSX.Element => (
    <div className="loading-indicator">
        loading...
    </div>
);

export default DefaultLoadingIndicator;
