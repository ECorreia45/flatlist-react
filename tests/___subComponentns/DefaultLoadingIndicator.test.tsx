import React from 'react';
import DefaultLoadIndicator from './../../src/___subComponents/DefaultLoadIndicator';
import {render} from '@testing-library/react';

describe('DefaultLoadIndicator', () => {
    it('Should match snapshot', () => {
        const {asFragment} = render(<DefaultLoadIndicator/>);

        expect(asFragment()).toMatchSnapshot();
    });
});
