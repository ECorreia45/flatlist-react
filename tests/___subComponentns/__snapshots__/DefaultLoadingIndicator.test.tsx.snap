// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultLoadIndicator Should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="loading-indicator"
  >
    loading...
  </div>
</DocumentFragment>
`;
